exp/addons/controllers/clusterresourceset_controller_test.go

```diff
@@ -242,10 +239,9 @@ metadata:
 		}, timeout).Should(BeTrue())
 	})
 	It("Should reconcile a ClusterResourceSet when a resource is created that is part of ClusterResourceSet resources", func() {
-		labels := map[string]string{"foo2": "bar2"}
-		newCMName := "test-configmap3"
+		newCMName := fmt.Sprintf("test-configmap-%s", util.RandomString(6))
```

Comment:
You could use the metadata `GenerateName` to do this for you on create, though I guess that means you have to create the configmap before referencing it elsewhere which may make the tests more complex

Reply:
Right, I prefer to keep it as is.